roles/horizon/templates/etc/openstack-dashboard/local_settings.py

```diff
@@ -180,6 +180,25 @@
 # of your entire OpenStack installation, and hopefully be in UTC.
 TIME_ZONE = "UTC"
 
+#
+# The Horizon Policy Enforcement engine uses these values to load per service
+# policy rule files. The content of these files should match the files the
+# OpenStack services are using to determine role based access control in the
+# target installation.
+
+# Path to directory containing policy.json files
+POLICY_FILES_PATH = "/opt/stack/horizon/conf"
+# Map of local copy of service policy files
+POLICY_FILES = {
+    'identity': 'keystone_policy.json',
+    'compute': 'nova_policy.json',
+    'volume': 'cinder_policy.json',
+    'image': 'glance_policy.json',
+    'orchestration': 'heat_policy.json',
+    'network': 'neutron_policy.json',
+    'telemetry': 'ceilometer_policy.json',
+}
```

Comment:
So originally, I was following the documentation (http://docs.openstack.org/developer/horizon/topics/policy.html), but i wasn't sure if we'd want to override the default location, e.g. map policy files like 'identity': '/etc/keystone/policy.json'.

But you made a good point about doing it all in local settings. I'm going to try that, confirm it doesn't break anything, then commit those changes, should remove the need for any ursula changes, and keep all changes in local settings.